src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/Tickers.kt

```diff
@@ -58,57 +78,87 @@ data class SupportedTickersParameters(
      */
-    val search: String? = null,
+    val cusip: String? = null,
+
+    /**
+     * Specify the CIK of the asset you want to search for.
+     * Find more information about CIK codes at their website.
+     * Defaults to empty string which queries all CIKs.
+     */
+    val cik: String? = null,
 
     /**
-     * How many items to be on each page during pagination. Max: 50, defaults to 25.
+     * Specify a point in time to retrieve tickers available on that date.
+     * Defaults to the most recent available date.
      */
-    @DefaultValue("25")
-    val tickersPerPage: Int = 25,
+    val date: String? = null,
 
     /**
-     * The page of results to return. Defaults to 1.
+     * (Optional) Search the name of tickers (ex: "microsoft")
      */
-    @DefaultValue("1")
-    val page: Int = 1,
+    val search: String? = null,
 
     /**
      * (Optional) Filter for only active or inactive symbols.
      * If not set, returns both active and inactive symbols
      */
-    val activeSymbolsOnly: Boolean? = null
+    val activeSymbolsOnly: Boolean? = null,
+
+    /**
+     * Which field to sort by (ex: "ticker", "type", etc). Defaults to "ticker"
+     */
+    @DefaultValue("ticker")
+    val sortBy: String = "ticker",
+
+    /**
+     * Whether or not to sort ascending (A-Z) or descending (Z-A). Defaults to false
+     */
+    @DefaultValue("false")
+    val sortDescending: Boolean = false,
+
+    /**
+     * Limit the size of the response, default is 100 and max is 1000.
+     *
+     * If your query returns more than the max limit and you want to retrieve the next page of results,
+     * see the next_url response attribute.
+     */
+    @DefaultValue("10")
+    val limit: Int = 10
+
 )
 
 @Serializable
 data class TickersDTO(
     val status: String? = null,
-    val page: Int,
-    @SerialName("perPage") val tickersPerPage: Int? = null,
-    val count: Int,
-    val tickers: List<TickerDTO> = emptyList()
+    val count: Int? = null,
+    @SerialName("next_url") val nextUrl: String? = null,
+    @SerialName("request_id") val requestId: String? = null,
+    @SerialName("results") val results: List<TickerDTO>? = null
 )
 
 @Serializable
 data class TickerDTO(
-    val ticker: String? = null,
-    val name: String? = null,
-    val market: String? = null,
-    val locale: String? = null,
-    val type: String? = null,
-    val currency: String? = null,
     val active: Boolean = false,
-    @SerialName("primaryExch") val primaryExchange: String? = null,
-    @SerialName("updated") val lastUpdated: String? = null,
-    val url: String? = null,
-    val codes: Map<String, String> = emptyMap(),
-    val attrs: Map<String, String> = emptyMap()
+    val cik: String? = null,
+    @SerialName("composite_figi") val compositeFigi: String? = null,
+    @SerialName("currency_name") val currencyName: String? = null,
+    @SerialName("last_updated_utc") val lastUpdatedUtc: String? = null,
```

Comment:
Looks like we're missing the `delisted_utc` field in the DTO

```suggestion
    @SerialName("last_updated_utc") val lastUpdatedUtc: String? = null,
    @SerialName("delisted_utc") val delistedUtc: String? = null,
```